json_serializable/lib/src/builder.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:build/build.dart';
+import 'package:logging/logging.dart';
+
+import 'json_part_builder.dart';
+
+// TODO: until we can use `log` here - github.com/dart-lang/build/issues/1223
+final _logger = new Logger('json_serializable');
+
+/// Supports `package:build_runner` creation and configuration of `build_cli`.
+///
+/// Not meant to be invoked by hand-authored code.
```

Comment:
which is why I like to have it in a separate library 😁

Reply:
Hrm...k